diff -urN spandsp-0.0.6/configure.ac spandsp-0.0.6-new/configure.ac
--- spandsp-0.0.6/configure.ac	2014-06-04 16:27:54.000000000 +0200
+++ spandsp-0.0.6-new/configure.ac	2019-09-02 22:57:22.346611038 +0200
@@ -152,9 +152,9 @@
 
 AC_FUNC_ERROR_AT_LINE
 AC_FUNC_VPRINTF
-AC_FUNC_MALLOC
+#AC_FUNC_MALLOC
 AC_FUNC_MEMCMP
-AC_FUNC_REALLOC
+#AC_FUNC_REALLOC
 AC_FUNC_SELECT_ARGTYPES
 
 AX_C99_FUNC_LRINT
@@ -368,6 +368,7 @@
         ;;
     mingw*)
         COMP_VENDOR_LDFLAGS="-no-undefined -lws2_32"
+        COMP_VENDOR_CFLAGS="-DLIBSPANDSP_EXPORTS $COMP_VENDOR_CFLAGS"
         ;;
     *)
         COMP_VENDOR_LDFLAGS=
diff -urN spandsp-0.0.6/src/t4_rx.c spandsp-0.0.6-new/src/t4_rx.c
--- spandsp-0.0.6/src/t4_rx.c	2013-01-27 10:19:12.000000000 +0100
+++ spandsp-0.0.6-new/src/t4_rx.c	2019-09-04 22:12:42.620970198 +0200
@@ -103,6 +103,10 @@
 #include "spandsp/private/t4_rx.h"
 #include "spandsp/private/t4_tx.h"
 
+#ifdef __WIN32
+#include <winsock2.h>
+#endif
+
 /*! The number of centimetres in one inch */
 #define CM_PER_INCH                 2.54f
 
